docs/source/user_guide/installation/install-source-windows.rst

```diff
@@ -0,0 +1,382 @@
+        .. image:: images-windows-source/pic2-env-var-open.png
+			:scale: 70%
+
+
+    * Select ``Path`` from ``user variables`` if you want to set VCPKG just for you, or use ``Path`` from ``system variables`` to set VCPKG for all your system users.
+    	.. image:: images-windows-source/pic3-show-path.png
+		    :scale: 50%
+
+
+    * Select ``Path`` and select ``NEW``. Paste the location to VCPKG, then confirm your options by clicking ``OK``.
+    	.. image:: images-windows-source/pic4-save-env-var.png
+		  :height: 900px
+		  :width: 900px
+		  :scale: 70%
+
+
+5. Close and reopen Command Prompt and execute ``vcpkg`` to see an output of different options available for ``vcpkg``.
+
+CMake
+~~~~~
+
+``CMake`` is required to install the SUNDIALS and other libraries for the ``IDAKLU`` solver.
+To install it, follow the link to the `official CMake downloads page <https://cmake.org/download/>`_.
+
+Download an installer based on your system's architecture, i.e. ``x32/x64``, and check ``Add CMake to the PATH environment variable`` during installation.
+
+Python Packages
+~~~~~~~~~~~~~~~
+
+In addition to the packages above, you will need `TOML Kit <https://tomlkit.readthedocs.io/en/latest/>`_, a ``toml`` table parser. The tool is necessary to install build-time dependencies in "editable mode" without build isolation.
+
+To install ``tomlkit`` to your local user account (ensure you are not within a virtual environment), use the following command:
+
+.. code:: bash
+
+	  python3.X -m pip install --user tomlkit
+
+Finally, we recommend using `Nox <https://nox.thea.codes/en/stable/>`_.
+You can install it to your local user account (make sure you are not within a virtual environment) with
+
+.. code:: bash
+
+	  python3.X -m pip install --user nox
+
+Note that running ``nox`` will create new virtual environments for you to use, so you do not need to create one yourself.
+
+After installing the above packages, you must add the following location to your ``Path`` environment variable to run ``nox`` in a terminal, like Command Prompt.
+Follow the instructions above for VCPKG to add it to the ``Path``.
+
+.. code::
+
+	C:\Users\<USERNAME>\AppData\Roaming\Python\Python3<X>\Scripts
+
+Make sure to replace ``<USERNAME>`` with your user name and ``X`` with your Python subversion.
+
+.. _installing-pybamm:
+
+Installing PyBaMM
+-----------------
+
+Open a Command Prompt and navigate to the folder where you want to install PyBaMM,
+
+1. Obtain the PyBaMM source code by cloning the GitHub repository or downloading the source archive on the repository's homepage.
+
+   .. code:: bash
+
+		  git clone https://github.com/pybamm-team/PyBaMM.git
+
+2. Set the following environment variables by pasting the commands to your Command Prompt.
+
+   .. code-block:: bash
+
+		  set PYBAMM_USE_VCPKG=ON
+		  set VCPKG_ROOT_DIR=C:\vcpkg
+		  set VCPKG_DEFAULT_TRIPLET=x64-windows-static-md
+		  set VCPKG_FEATURE_FLAGS=manifests,registries
```

Comment:
It would be nice if we could set these as part of the IDAKLU install script instead of making the user set them

Reply:
We can do that by adding defaults to CMakeLists.txt. Apart from others, I will try to set `VCPKG_ROOT` (I will change VCPKG_ROOT_DIR to this) so we can use that if this is defined. Otherwise, we fall back to default.

Reply:
I tried doing that but to no avail. The next best thing I could think of was writing a `.bat` script which can permanently set environment variables.